prerequisites.rst

```diff
@@ -1,35 +1,250 @@
+  cannot officially be used for shared resources or as data drive**.
+  What supported brands and sizes are covered in the software requirements section below.
+  The recommendation for the System drive(s) hardware could be older spinning
+  Hard Disk Drives (HDD), Disk-on-Module [3]_, CompactFlash [4]_ or thumb drives (USB),
+  the Solid State Disks (SDD [2]_) must be managed.
+
+Data drive storage
+  The storage disk drive(s) where the data for the defined or used shares resources
+  wil be stored. **Cannot be the same of the system drive(s) and sizes will
+  depends of the usage of the resources**. Shared resources are managed over
+  partitions, which is discussed in the software requirements topic below.
+  The recommendation for the Data drives must be Solid State Disks (SSD [2]_ disks)
+  for best performance or spinning Hard Disk Drives (HDD).
+
+The memory (RAM)
+^^^^^^^^^^^^^^^^
+
+Enough RAM is vital to maintaining peak performance. There are several combinations
+of installation sizes but the **recommendation is at least 4Gib for novice administrators**.
+
+For best practice you should have 8GiB of RAM for basic operations in default installation.
+and no matter the size, RAM must be configured in Dual Channel mode [8]_ 
+inclusivelly if there are little amount of, it will improve performance.
+
+Unless other NAS systems the |omv| system can run in at least 1Gib of RAM, but
+of course will need expertise.
+
+The communication card (NIC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A NAS system as means is a network oriented system, of course |omv| is the
+most flexible NAS system in the world and allows multiple ways of communication, 
+this is becouse |omv| offers several modes of shared resources like RSYNC that 
+do not depends on networking. But there are few key points:
+
+- The NAS write speed to storage is limited to the speed of your NAS computer NIC card
+- The NAS write speed to storage is limited to the speed of your network
+- Speed of writing to the storage depends on ratio of the number of users accessing it.
+
+Full-duplex NICs are recommended, half-duplex NICs typically offer 10/100 Mbit/s
+and storage drives far exceed those speeds but are of course supported by |omv|.
+
+Any supported NIC vendor is determined by the support of the operating system
+where |omv| system resides and runs, which is Debian GNU/Linux.
+
+The architecture (CPU)
+^^^^^^^^^^^^^^^^^^^^^^
+
+We have already denoted in the requirements table the supported architectures or
+processors (which reduces and determines which computers can be used) but this is
+really a reference since it is determined by the support of the operating system
+where |omv| system resides and runs, which is Debian GNU/Linux.
+
+Any computer system supported by the common requirements of Debian operating
+system could be an |omv| system installation target. **Currently we supports
+amd64 (x66 64bits), i386 (x86 32bits), ARMv7+ARMv8 (arm64/armel/armhf)**.
+
```

Comment:
Add a warning that 32bit works, but with OMV6 32bit support has been stopped officially. It should work, but it is not tested nor new features take care that they might not work on 32bit systems.

Reply:
also fix amd64 x64

Reply:
thanks .. got catch @PrplHaz4  i was in doubt about it but after a search i used x86..  fixed in commit be624bd8da52ad54039c7533a818c8b7073b30b2